### ISSUE TITLE
ERROR! 'nxos_command' is not a valid attribute for a Play

### ISSUE THREAD

**Issue by rdsmf** [networktocode/ntc-ansible#171]
I am at a lost on this but I am trying to write a playbook to test against my Nexus 9K swithces in my lab, but I keep getting this error "ERROR! 'nxos_command' is not a valid attribute for a Play". The back drop story on this is I did a update to Ansible from 2.2 to 2.3 using PIP and I can not seem to get past this syntax error. Can anyone look at my playbook and tell me what I am doing wrong? Here is my playbook.....

---
- vars:
    cli:
      host: "{{ inventory_hostname }}"
      username: admin
      password: admin
      transport: cli

- name: run show version on remote devices
  nxos_command:
      commands: show version
      provider: "{{ cli }}"

...

**Comment by jedelman8** [comment]
Try using backticks when posting a playbook.  While this is not related to this repo (ntc-ansible), it seems to be an easy one.

You don't have a play definition such as:

```
- name: PLAY
  hosts: all
  connection: local
  gather_facts: no
```

and you don't have `tasks:` 

If you find any sample playbook, you should see what I mean.

**Comment by rdsmf** [comment]
Thank you for the suggestions, I tried to run the playbook and I am getting "ERROR! Syntax Error while loading YAML". I am using nano in Ubuntu to edit my playbooks, is this editor ok or is there another editor I should be using to create playbooks? Here is the updated playbook I did.....

---

- name: TEST PLAYBOOK
  hosts: all
  connection: local
  gather_facts: no

vars:
  nxapi:
    host: `{{ inventory_hostname }}`
    username: admin
    password: admin
    transport: nxapi
    use_ssl: no

tasks:
  - name: run show version
    nxos_command:
      commands: show version
      provider: `{{ nxapi }}`
